File: PYTHON/tpl-rand-0001.py
#!/usr/bin/env python

# --- TEXTPATGEN-QUICKSCRIPT-PYTHON ---

# This QuickScript generates random data.

# Users can change the output by editing
# this file directly.

# The text is written to a timestamped file.

import time
import random
from random import randrange

fp=open(time.strftime('00_%Y%m%d_%H%M%S.txt'), 'w')
fp.write(time.strftime('TEXTPATGEN-QUICKSCRIPT-PYTHON: %Y-%m-%dT%H:%M:%S%z\n'))
random.seed(100)
for length in range(0, 16):
  for width in range(0, 15):
    num=randrange(0, 65535)
    fp.write('X-%04X ' % num)
    width=width+1
    length=length+1
  num=randrange(0, 65535)
  fp.write('X-%04X\n' % num)
fp.close()

File: LUA/tpl-rand-0001.lua
-- TEXTPATGEN-QUICKSCRIPT-LUA
-- Please edit this file to suit your preferences.
-- The text is written to a timestamped file.

filename = os.date("00_%Y%m%d_%H%M%S.txt")
file = io.open(filename, "w")
introstr = os.date("TEXTPATGEN-QUICKSCRIPT-LUA: %Y-%m-%dT%H:%M:%S%z\n")
file:write(introstr)
math.randomseed(100)
n3 = math.random(0, 65535)
for n1 = 0, 15 do
  for n2 = 0, 14 do
    str = string.format("X-%04X ", n3)
    file:write(str)
    n3 = math.random(0, 65535)
    end 
  str = string.format("X-%04X\n", n3)
  file:write(str)
  n3 = math.random(0, 65535)
end
file:close()

